shrinker-test/src/main/java/com/example/NoSerializedNameMain.java

```diff
@@ -19,25 +19,25 @@ static class TestClassNotAbstract {
     public String s;
   }
 
-  static class TestClassWithoutDefaultConstructor {
+  static class TestClassConstructorHasArgs {
     public String s;
 
     // Specify explicit constructor with args to remove implicit no-args default constructor
-    public TestClassWithoutDefaultConstructor(String s) {
+    public TestClassConstructorHasArgs(String s) {
       this.s = s;
     }
   }
 
   /**
-   * Main entrypoint, called by {@code ShrinkingIT.testNoSerializedName_DefaultConstructor()}.
+   * Main entrypoint, called by {@code ShrinkingIT.testNoSerializedName_ConstructorHasArgs()}.
```

Comment:
I think this should be
```suggestion
   * Main entrypoint, called by {@code ShrinkingIT.testNoSerializedName_NoArgsConstructor()}.
```